src/azure-cli/setup.py

```diff
@@ -126,7 +126,7 @@
     'azure-storage-common~=1.4',
     'azure-synapse-accesscontrol~=0.2.0',
     'azure-synapse-spark~=0.2.0',
-    'cryptography>=2.3.1,<3.0.0',
+    'cryptography>=2.3.1',
```

Comment:
normal practice in CLI is pin exact major version. what's impact of the change?

Reply:
It might be normal practice here, but I'm afraid it's extremely bad practice to randomly pin to whatever is available that day of the week, without any specific reason. There is no incompatibility introduced, so it's unnecessary. The impact is that python3-cryptography is moving on everywhere, and the old deprecated versions are not available anymore, so we have to manually patch this downstream.

https://packages.debian.org/search?keywords=python3-cryptography&searchon=names&suite=all&section=all
https://packages.ubuntu.com/search?keywords=python3-cryptography&searchon=names&suite=all&section=all
https://rpmfind.net/linux/rpm2html/search.php?query=python3-cryptography